cloud/services/networking/dns.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Maybe we can move this file to another directory like `utils` in the future. But leave it as is for now